inspire/modules/authors/templates/authors/author_stub.html

```diff
@@ -0,0 +1,29 @@
+{# -*- coding: utf-8 -*-
+##
+## This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- extends 'page.html' -%}
+
+{%- block body -%}
+  <div class="alert alert-warning">
+    {{ _("You tried to access an author module page.
+    Author module is currently disabled. A new version
+    is under development and will be released soon.") }}
```

Comment:
Fair point.

Unfortunately, it doesn't work, and I will look for a solution in the future.